### DISCUSSION TITLE
Readme usable by noobs

### DISCUSSION THREAD

**Discussion by Clay-Ferguson** [undyingwraith/ipsm#14] [Ideas]
I would make the README file be so simple that all anyone needs to know is how to unzip the project, and run commands. 

Assume they know nothing at all about yarn. Say exactly which directory to change into, and exactly what command to run. You've already almost got this but I had trouble before realizing I need needed to `cd packages/web`. Don't assume anyone knows anything at all. People who know nothing but JavaScript should hopefully be able to play with this code.

**Comment by undyingwraith** [comment]
Yeah I already updated the README a bit. So it should be easier to get started. But any help in making it idiot proof is nice too.

BTW: I plan on publishing this on ipfs, so no need to mess with the repo. But for possible collaborators this would still be nice.

**Reply by Clay-Ferguson** [reply]
Great idea! Having the actual app itself come up right in the browser via CID will be really slick!

**Reply by undyingwraith** [reply]
Current problem is that the way i tried it the gateway didnt set the correct mime type headers so the browser rejected the js and css files. I got other apps working before, idk thats a thing for later. RN i want the basics to work and be kinda solid.

**Reply by Clay-Ferguson** [reply]
I'm not that experienced either with adding folders to IPFS like via command line. I think MFS has mime type, but no clue what might have stopped it from working.

**Reply by undyingwraith** [reply]
For reference: `ipfs add -rQt build` worked

You can check it out here: https://k51qzi5uqu5dj00g8grr9pby14wibhuhkgxo7gdb5ebnlem2eaeob3809qni8p.ipns.dweb.link

**Reply by Clay-Ferguson** [reply]
Nice! That would be a great thing to add to the readme so people can see a sample working prototype of the app right away.  Also they won't know they need to have IPFS 5001 port running locally either, so maybe a readme a sentence for that?

Also, even though it's still just bare bones prototype, a screenshot of the app full of messages in the readme will go a long way in getting people interested, and so also a readme section with just 3 or 4 sentences explaining how to use the app?
